Cargo.lock

```diff
@@ -93,7 +93,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "77c3a9648d43b9cd48db467b3f87fdd6e146bcc88ab0180006cef2179fe11d01"
 dependencies = [
  "cfg-if",
- "getrandom 0.2.12",
+ "getrandom 0.2.11",
```

Comment:
I don't think this was intentional?

Reply:
It wasn't, thanks: https://github.com/paradigmxyz/reth/pull/6003